fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java

```diff
@@ -110,6 +110,7 @@ public LoanDTO populateLoanDtoFromMap(final Map<String, Object> accountingBridge
         boolean isAccountTransfer = (Boolean) accountingBridgeData.get("isAccountTransfer");
         boolean isLoanMarkedAsChargeOff = (Boolean) accountingBridgeData.get("isChargeOff");
         boolean isLoanMarkedAsFraud = (Boolean) accountingBridgeData.get("isFraud");
+        BigDecimal overpaidAmount = (BigDecimal) accountingBridgeData.get("overpaidAmount");
```

Comment:
You dont need this, LoanTransactionDTO contains the overpaid amount

Reply:
removed it